demo/vl-input-field.html

```diff
@@ -82,8 +82,59 @@ <h2>Input-field small</h2>
       </div>
   </div>
+
+
+    <div>
+        <h2>Formulier validatie</h2>
+        <pre>
+            <code class="line-numbers language-markup">
+                &lt;form&gt;
+                        &lt;div&gt;
+                            &lt;label for="input-voornaam" block&gt;Voornaam&lt;/label&gt;
+                            &lt;input id="input-voornaam" is='vl-input-field' placeholder="Jos" block data-required="true"
+                                data-vl-error-message="Veld &quot;Voornaam&quot; is verplicht"
+                                data-vl-error-placeholder="validation-voornaam-error" /&gt;
+                            &lt;p data-vl-error-id="validation-voornaam-error"&gt;&lt;/p&gt;
+                        &lt;/div&gt;
+                        &lt;div&gt;
+                            &lt;label is="vl-form-label" for="input-iban" block&gt;IBAN-nummer&lt;/label&gt;
+                            &lt;input id="input-iban" is='vl-input-field' placeholder="BE00 0000 0000 0000" block type="text"
+                                data-required="true" 
+                                data-vl-error-message="Een geldig &quot;IBAN-nummer&quot; is verplicht"
+                                data-vl-error-placeholder="validation-iban-error" data-vl-validation-type="iban" /&gt;
+                            &lt;p data-vl-error-id="validation-iban-error"&gt;&lt;/p&gt;
+                        &lt;/div&gt;
+                        &lt;div&gt;
+                            &lt;button&gt;Formulier valideren&lt;/button&gt;
+                       &lt;/div&gt;
+                &lt;/form&gt;
+            </code>
+        </pre>
+        <div class="demo">
+            <form>
+                    <div>
+                        <label for="input-voornaam" block>Voornaam</label>
+                        <input id="input-voornaam" is='vl-input-field' placeholder="Jos" block data-required="true"
+                            data-vl-error-message="Veld &quot;Voornaam&quot; is verplicht"
+                            data-vl-error-placeholder="validation-voornaam-error" />
+                        <p id="validation-voornaam-error-message" data-vl-error-id="validation-voornaam-error"></p>
+                    </div>
+                    <div>
+                        <label for="input-iban" block>IBAN-nummer</label>
+                        <input id="input-iban" is='vl-input-field' placeholder="BE00 0000 0000 0000" block type="text"
+                            data-required="true" data-vl-error-message="Een geldig &quot;IBAN-nummer&quot; is verplicht"
+                            data-vl-error-placeholder="validation-iban-error" data-vl-validation-type="iban" />
+                        <p id="validation-iban-error-message" data-vl-error-id="validation-iban-error"></p>
+                    </div>
+                    <div>
+                        <button id="form-validate-button">Formulier valideren</button>
```

Comment:
Ik zou de vl-ui-button durven includen als dev dependency, dit zal vermoedelijk nooit een cyclic dependency veroorzaken. Het geeft de demo pagina wat sexyness en is dan ook analoog aan de andere demo pagina's.

Reply:
voorlopig niet. eerst nog eens kijken naar de dependency tree.